src/main/java/com/exasol/dbbuilder/dialects/exasol/AdapterScript.java

```diff
@@ -27,6 +30,33 @@ public AdapterScript(final ExasolImmediateDatabaseObjectWriter writer, final Sch
         this.writer = writer;
         this.language = language;
         this.content = content;
+        this.debuggerConnection = Optional.empty();
+        this.writer.write(this);
+        if (isAdapterScriptDebuggingEnabled()) {
+            throw new IllegalStateException("Adapter script debugging was enabled but no debugger address was passed."
+                    + "Please use a different constructor.");
+        }
+    }
+
+    /**
+     * Create a new instance of an {@link AdapterScript} with possible debugger connection. The debugger is only
+     * attached if the property test.debugAdapterScripts="true" is set. You can set it by appending -Dtest.debug="true"
+     * to your the JVM.
```

Comment:
```suggestion
    /**
     * Create a new instance of an {@link AdapterScript} with possible debugger connection.
     * <p> The debugger is only
     * attached if the property test.debugAdapterScripts="true" is set. You can set it by appending -Dtest.debug="true"
     * to your the JVM.
     * </p>
```

Please remember that JavaDoc does not handle paragraphs automatically. You have to separate the title of a JavaDoc entry from the descriptions paragraphs by hand.